cargo-near/src/main.rs

```diff
@@ -67,6 +67,7 @@ fn main() -> CliResult {
     } else {
         env::args().next().unwrap_or("./cargo".to_string())
     };
+    export_nep_330_build_command();
```

Comment:
I don't feel that it is the right place for it. Can we move it to the build command implementation?